### PR TITLE
Alphabetically arranged the MODULES.md file #1167 - Sorted index and the corpus

### LINKED ISSUE #1167: Alphabetically arrange modules in Module docs
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

_**To gain a sense of what the issue targets go to [MODULES.md](https://github.com/publiclab/image-sequencer/blob/main/docs/MODULES.md) , we see the module names are arranged somewhat randomly. So the task is to arrange them to make it visibly easy to search for some specific module doc.**_

- [ ] 📝 **Update** the following file

   **1. [MODULES.md](https://github.com/publiclab/image-sequencer/blob/main/docs/MODULES.md) and arrange the module names' list alphabetically.**

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### Please show us where to look

https://beta.sequencer.publiclab.org

### What's your PublicLab.org username?

> This can help us diagnose the issue:

### Browser, version, and operating system

> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site.

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by kirito-k**
Fixes #1167 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

File: docs/MODULES.md [hunk 1]
```diff
@@ -3,118 +3,49 @@ Documentation of various Modules
...
+29. [Overlay](#overlay-module)
+30. [PaintBucket](#paint-bucket-module)
+31. [ReplaceColor](#replacecolor-module)
+32. [Resize](#resize-module)
+33. [Rotate](#rotate-module)
+34. [Saturation](#saturation-module)
+35. [Segmented-Colormap](#segmented-colormap-module)
+36. [Threshold](#threshold)
+37. [Tint](#tint)
 38. [Text-Overlay](#text-overlay)
```
Comment (aashna27) [inline docs/MODULES.md]: Hey your work is really great just a little change can you change the order for text overlay after segmented? Sorry for the trouble

**Comment by kirito-k** [comment]
Hey I did make changes and pushed them. Did you receive anything?

**Comment by aashna27** [comment]
Hey yeah cool, the changes exist!! Hooray great work

**Comment by aashna27** [comment]
@publiclab/reviewers let's have a look and approve this to get it merged at the earliest!

**Comment by aashna27** [comment]
I guess only @jywarren can merge this so once he comes next friday this pr can be prioritised to be merged first.

**Comment by kirito-k** [comment]
@aashna27 Thank you so much. :)

**Comment by aashna27** [comment]
@jywarren this can be merged.

**Comment by jywarren** [comment]
Great work! Thanks so much!!!

**Pull request merged**